source/db/en-projects.json

```diff
@@ -562,17 +562,18 @@
   },
   {
     "development_stage": "released",
-    "description": "OTR-encrypted IM for Android and iOS.",
```

Comment:
I think this looks better:

> OMEMO- or OTR-encrypted IM for iOS

but feel free to disagree. Also, please remove now-outdated localizations from other languages.